src/vs/workbench/contrib/mergeEditor/browser/mergeEditorInputModel.ts

```diff
@@ -191,41 +191,49 @@ class TempFileMergeEditorInputModel extends EditorModel implements IMergeEditorI
 
 			const hasUnhandledConflicts = inputModels.some((m) => m.model.hasUnhandledConflicts.get());
 
-			const options: IDialogOptions = {
-				cancelId: 2,
+			const buttons: IPromptButton<ConfirmResult>[] = [
```

Comment:
@hediet fyi please check, this one is a bit more complex adoption of the new dialog api